src/main/java/TaskManager.java

```diff
@@ -0,0 +1,84 @@
+import java.util.ArrayList;
+
+public class TaskManager {
+
+    public static final String ERROR_TASK_NUMBER = "Invalid task number!";
+    public static final String BLANK_COMMAND = "Command Should not be blank!";
+    public static final String ADD_TASK_MESSAGE = "Got it. I've added this task:";
+    public static final String TASK_ALREADY_COMPLETED = "Task already marked as completed!";
+    public static final String DEADLINE_DELIMITER = "/by";
+    public static final String EVENT_DELIMITER = "/at";
+    public static final String TASK_DONE_MESSAGE = "Nice! I've marked this task as done:";
+
+    private ArrayList<Task> taskList;
+
+    public TaskManager() {
+        taskList = new ArrayList<>();
+    }
+
+    /**
+     * Marks a task as done. Informs user that that task has been completed.
+     *
+     * @param index the index of task in the taskList
+     */
+    public void markTaskAsDone(UI UI, int index) {
+
+        if (index >= 0 && index < taskList.size()) {
```

Comment:
Perhaps you might want to have a method check for these conditions instead to make it more intuitive